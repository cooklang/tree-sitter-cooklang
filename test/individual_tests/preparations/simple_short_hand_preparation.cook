Mix @onion{1}(peeled and finely chopped) into paste.