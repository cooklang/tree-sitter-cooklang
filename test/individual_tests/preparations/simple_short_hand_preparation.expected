(recipe 
  (step 
    (plain_text)
    (ingredient 
      name: (ingredient_text) 
      (amount (quantity))
      note: )
    (plain_text)))